test/jdk/sun/security/provider/acvp/ML_DSA_Intrinsic_Test.java

```diff
@@ -0,0 +1,517 @@
+
+            -3930395, -3930395, -1528703, -1528703, -3677745, -3677745, -3041255, -3041255,
+            -1452451, -1452451, 3475950, 3475950, 2176455, 2176455, -1585221, -1585221,
+            -1257611, -1257611, 1939314, 1939314, -4083598, -4083598, -1000202, -1000202,
+            -3190144, -3190144, -3157330, -3157330, -3632928, -3632928, 126922, 126922,
+            3412210, 3412210, -983419, -983419, 2147896, 2147896, 2715295, 2715295,
+            -2967645, -2967645, -3693493, -3693493, -411027, -411027, -2477047, -2477047,
+            -671102, -671102, -1228525, -1228525, -22981, -22981, -1308169, -1308169,
+            -381987, -381987, 1349076, 1349076, 1852771, 1852771, -1430430, -1430430,
+            -3343383, -3343383, 264944, 264944, 508951, 508951, 3097992, 3097992,
+            44288, 44288, -1100098, -1100098, 904516, 904516, 3958618, 3958618,
+            -3724342, -3724342, -8578, -8578, 1653064, 1653064, -3249728, -3249728,
+            2389356, 2389356, -210977, -210977, 759969, 759969, -1316856, -1316856,
+            189548, 189548, -3553272, -3553272, 3159746, 3159746, -1851402, -1851402,
+            -2409325, -2409325, -177440, -177440, 1315589, 1315589, 1341330, 1341330,
+            1285669, 1285669, -1584928, -1584928, -812732, -812732, -1439742, -1439742,
+            -3019102, -3019102, -3881060, -3881060, -3628969, -3628969, 3839961, 3839961,
+
+            2091667, 3407706, 2316500, 3817976, -3342478, 2244091, -2446433, -3562462,
+            266997, 2434439, -1235728, 3513181, -3520352, -3759364, -1197226, -3193378,
+            900702, 1859098, 909542, 819034, 495491, -1613174, -43260, -522500,
+            -655327, -3122442, 2031748, 3207046, -3556995, -525098, -768622, -3595838,
+            342297, 286988, -2437823, 4108315, 3437287, -3342277, 1735879, 203044,
+            2842341, 2691481, -2590150, 1265009, 4055324, 1247620, 2486353, 1595974,
+            -3767016, 1250494, 2635921, -3548272, -2994039, 1869119, 1903435, -1050970,
+            -1333058, 1237275, -3318210, -1430225, -451100, 1312455, 3306115, -1962642,
+            -1279661, 1917081, -2546312, -1374803, 1500165, 777191, 2235880, 3406031,
+            -542412, -2831860, -1671176, -1846953, -2584293, -3724270, 594136, -3776993,
+            -2013608, 2432395, 2454455, -164721, 1957272, 3369112, 185531, -1207385,
+            -3183426, 162844, 1616392, 3014001, 810149, 1652634, -3694233, -1799107,
+            -3038916, 3523897, 3866901, 269760, 2213111, -975884, 1717735, 472078,
+            -426683, 1723600, -1803090, 1910376, -1667432, -1104333, -260646, -3833893,
+            -2939036, -2235985, -420899, -2286327, 183443, -976891, 1612842, -3545687,
+            -554416, 3919660, -48306, -1362209, 3937738, 1400424, -846154, 1976782
+    };
+}
+// java --add-opens java.base/sun.security.provider=ALL-UNNAMED  -XX:+UseDilithiumIntrinsics test/jdk/sun/security/provider/acvp/ML_DSA_Intrinsic_Test.java
```

Comment:
This is line is useful. Not sure I would hide it at the bottom of the file.

Reply:
I actually meant to delete it, but will move it to the top.